ingress/controllers/nginx/nginx/command.go

```diff
@@ -29,6 +29,14 @@ import (
 
 	"k8s.io/contrib/ingress/controllers/nginx/nginx/config"
 	"k8s.io/contrib/ingress/controllers/nginx/nginx/ingress"
+
+	"expvar"
+)
+
+// number of reload performed
+var (
+	reload = expvar.NewInt("reload")
```

Comment:
Ah so /debug/vars/ dumps all such variables as json or some k:v format I'm guessing. Thanks for the clarification. Please add a comment so someone doesn't refactor it out. It actually might make more sense to setup an explicit handler that just returns the required variables so the relationship is explicit, but I leave that to you since it's pretty minor.